package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "kontena-lens",
+  "name": "lens-app",
```

Comment:
We cannot change this (yet) because it breaks packaging.

Reply:
ok. didn't know about that. maybe it's OK to talk about kontena some more :) but I'd like to see some kind of exit strategy from this "dependency" at some point